src/test/java/org/opensearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -409,6 +405,51 @@ private void verifyPatch(final boolean sendAdminCert, Header... restAdminHeader)
         Assert.assertTrue(settings.get("nagilum.hash").equals(""));
     }
 
+    private void verifyAuthToken(final boolean sendAdminCert, Header... restAdminHeader) throws Exception {
+
+        // Add enabled service account then generate auth token
+
+        rh.sendAdminCertificate = sendAdminCert;
+        HttpResponse response = rh.executePutRequest(ENDPOINT + "/internalusers/happyServiceLive",
+                ENABLED_SERVICE_ACCOUNT_BODY, restAdminHeader);
+        Assert.assertEquals(response.getBody(), HttpStatus.SC_CREATED, response.getStatusCode());
+        rh.sendAdminCertificate = sendAdminCert;
+        response = rh.executeGetRequest(ENDPOINT + "/internalusers/happyServiceLive", restAdminHeader);
+        Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
+
+        response = rh.executeGetRequest(ENDPOINT + "/internalusers/happyServiceLive/authtoken",
+                ENABLED_SERVICE_ACCOUNT_BODY, restAdminHeader);
+        Assert.assertEquals(response.getBody(), HttpStatus.SC_CREATED, response.getStatusCode());
```

Comment:
So for asserts, you can provide 3 arguments in which case the first one is what will be output if the assertion fails. It would be the whole body though in this case.